build-system/tasks/e2e/expect.js

```diff
@@ -145,13 +145,13 @@ function overwriteAlwaysUseSuper(utils) {
       const obj = this._obj;
       const isControllerPromise = obj instanceof ControllerPromise;
       if (!isControllerPromise) {
-        return _super.apply(this, arguments);
+        return Promise.resolve(_super.apply(this, arguments));
```

Comment:
This is unnecessary, you're inside an async promise which will implicitly wrap the result.

Reply:
+1

Reply:
Ah thanks, that makes sense